src/one_dragon/utils/log_utils.py

```
@@ -1,39 +1,138 @@
+    configured = (log_file_path or '').strip()
+    if not configured:
+        configured = default_name
+    if os.path.isabs(configured):
+        return configured
+    return os.path.join(os_utils.get_path_under_work_dir('.log'), configured)
+
+
+def get_logger():
+    """获取框架默认 logger。
+
+    若尚未初始化，则按默认配置初始化一次；若已经存在框架默认 handler，则直接复用。
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)